sample/build.gradle.kts

```
@@ -1,11 +1,56 @@
 plugins {
+    kotlin("multiplatform")
     id("com.android.application")
-    kotlin("android")
     id("org.jmailen.kotlinter")
 }
 
 // TODO: When we add other platforms, this should become a multiplatform module.
 
+kotlin {
+    android { publishAllLibraryVariants() }
```

You're not publishing your sample app.

```suggestion
    android()
```